.github/workflows/cleanup-all.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         run: (cd .github/actions/cleanup-pe && go run .)
 
       - name: Run atlas project cleaner
+        if: always()
```

Comment:
Why this here?

Reply:
It will execute this job even even the previous ones failed